src/mscorlib/src/System/Runtime/Loader/AssemblyLoadContext.cs

```diff
@@ -57,7 +68,67 @@ protected AssemblyLoadContext()
         internal AssemblyLoadContext(bool fDummy)
         {
         }
-        
+#else // !FEATURE_COLLECTIBLE_ALC
+        [System.Security.SecuritySafeCritical]
+        protected AssemblyLoadContext()
+            : this(true) // TODO: Testing only, should default to false!
+        {
+        }
+
+        protected AssemblyLoadContext(bool isCollectible)
```

Comment:
model.xml accepts conditions the same way as we set up #ifdef's. However I don't really like public APIs be conditions so we should either take them or not. We should avoid having them sometimes and not other times.